src/h5web/vis-packs/core/shared/PanZoomMesh.tsx

```diff
@@ -94,16 +93,13 @@ function PanZoomMesh() {
     moveCameraTo(camera.position.x, camera.position.y);
   }, [camera, moveCameraTo]);
 
-  const wheelCaptureRef = useWheelCapture();
+  useWheelCapture();
 
   return (
-    <>
-      <mesh {...{ onPointerMove, onPointerUp, onPointerDown, onWheel }}>
-        <meshBasicMaterial opacity={0} transparent />
-        <planeGeometry args={[width, height]} />
-      </mesh>
-      <Html ref={wheelCaptureRef} />
```

Comment:
It stopped working because of the `pointer-events: none` set on the root DOM element rendered by the `Html` compoent. Now, I hook straight onto the canvas' `domElement`, so we no longer need an `Html` element here!